lib/src/excel.dart

```diff
@@ -126,6 +197,46 @@ class ExcelDart {
     }));
   }
 
+  /// Highlight rows that are underpacing.
+  ///
+  /// Structured reference (i.e. Table[@Column]) is not allowed in Excel
+  /// conditional formatting. Regular reference (i.e. $A5) has to be used
+  /// instead. As table expands, Excel would automatically extends the
+  /// conditional formatting rule to cover the entire range.
+  static void _highlightUnderpacingRows(String tableName) async {
+    await ExcelJS.run(allowInterop((context) async {
+      final table = context.workbook.tables.getItem(tableName);
+      final range = table.getDataBodyRange();
+      final format = range.conditionalFormats.add('Custom');
+
+      // Excel formula DATEDIF(dateA, dateB, "D") calculates the difference
+      // between two dates in days, where dateB must be after or equal to dateA.
+      // i.e. DATEDIF("7/1/2020", "7/31/2020", "D") = 30
+      //
+      // This formula calculates:
+      // (spent / budget) / (current duration / flight duration)
+      //
+      // Flight duration is calculated by (end date - start date + 1), and
+      // current duration is calculated by (now - start date + 1), so it is
+      // guaranteed that divide by zero will not happen.
+      final formula = ''' 
+       ($_spentColumn/$_budgetColumn) / 
```

Comment:
not sure about this formula. lets drop an issue here and immediately follow up with another pr and get arul's eyes on it. I think adding to the numerator and denominator will change the value. Lets get better granularity than days on the date calc, and not return answers when it has been in flight for <2 days or <10% of its flight (for e.g.) to guard against weird edge cases.